src/internal/packager/git/pull.go

```
@@ -27,7 +27,7 @@ func (g *Git) DownloadRepoToTemp(gitURL string) string {
 	// packaged anyway, and it saves us from having to fetch the tags
 	// later if we need them
 
-	err = g.pull(gitURL, path, "")
+	_ = g.pull(gitURL, path, "")
```

Can we add a comment as to why we ignore this error

---

I actually dont know why we are.........

---

Yeah I think DownloadRepo to temp is used by our helm chart logic only. We fatalf above, could we fatalf here too?

---

we've been trying to remove all the fatals from the git package actually

---

Then we can refactor it to return an error instead potentially.  Or if we want to do that in another PR add a TODO comment.